src/clientiface.cpp

```diff
@@ -228,7 +228,7 @@ void RemoteClient::GetNextBlocks (
 		wanted_range);
 	const s16 d_cull_opt = std::min(adjustDist(m_block_cull_optimize_distance, prop_zoom_fov),
 		wanted_range);
-	const s16 d_blocks_in_sight = full_d_max * BS * MAP_BLOCKSIZE;
+	const f32 d_blocks_in_sight = full_d_max * BS * MAP_BLOCKSIZE;
```

Comment:
Perhaps you could add a short comment, so that it isn't changed back to s16 by accident?